python/cudf/cudf/core/series.py

```diff
@@ -1000,7 +1000,13 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
         if fname in binary_operations:
             not_reflect = self is inputs[0]
             other = inputs[not_reflect]
-            op = f"__{'' if not_reflect else 'r'}{binary_operations[fname]}__"
+            op = binary_operations[fname]
+            ops_without_reflection = ("gt", "ge", "lt", "le", "ne", "eq")
+            if op in ops_without_reflection and not not_reflect:
+                # This replacement will ignore eq and ne, which don't need it.
+                op = op.replace(*(("g", "l") if "g" in op else ("l", "g")))
```

Comment:
An alternative way is to write `ops_without_refection` as a dictionary mapping the op to its inverse op. Possibly clearer in intent without need of the comment?

Reply:
Yeah I did that originally, then switched to this more brief version. I can switch back to that.